src/shared_utilities/drivers.jl

```diff
@@ -1642,6 +1688,194 @@ function prescribed_forcing_era5(
     return (; atmos, radiation)
 end
 
+"""
+     prescribed_perturbed_forcing_era5(era5_ncdata_path,
+                             surface_space,
+                             start_date,
+                             earth_param_set,
+                             ΔT,
+                             FT;
+                             gustiness=1,
+                             max_wind_speed = nothing,
+                             c_co2 = TimeVaryingInput((t) -> 4.2e-4),
+                             time_interpolation_method = LinearInterpolation(PeriodicCalendar()),
+                             regridder_type = :InterpolationsRegridder,
+                             interpolation_method = Interpolations.Constant(),)
+
+A helper function which constructs the `PrescribedAtmosphere` and `PrescribedRadiativeFluxes`
+from a file path pointing to the ERA5 data in a netcdf file, the surface_space, the start date,
+and the earth_param_set, applying a change in the instantaneous temperature at each point
+of ΔT.
+
+Please see the documentation for `prescribed_forcing_era5` for more information.
+"""
+function prescribed_perturbed_forcing_era5(
+    era5_ncdata_path,
+    surface_space,
+    start_date,
+    earth_param_set,
+    ΔT,
+    FT;
+    gustiness = 1,
+    max_wind_speed = nothing,
+    c_co2 = TimeVaryingInput((t) -> 4.2e-4),
+    time_interpolation_method = LinearInterpolation(PeriodicCalendar()),
+    regridder_type = :InterpolationsRegridder,
+    interpolation_method = Interpolations.Constant(),
+)
+    # Pass a list of files in all cases
+    era5_ncdata_path isa String && (era5_ncdata_path = [era5_ncdata_path])
+
+    # Precip is provide as a mass flux; convert to volume flux of liquid water with ρ =1000 kg/m^3
+    precip = TimeVaryingInput(
+        [era5_ncdata_path, era5_ncdata_path],
+        ["mtpr", "msr"],
+        surface_space;
+        start_date,
+        regridder_type,
+        regridder_kwargs = (; interpolation_method),
+        file_reader_kwargs = (; preprocess_func = (data) -> -data / 1000,),
```

Comment:
```suggestion
        file_reader_kwargs = (; preprocess_func = (data) -> -data / ρ_water,),
```
(with the density read from parameters, to avoid magic numbers here)